packages/data-seeder/src/locations.ts

```diff
@@ -9,46 +9,49 @@
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
 import fetch from 'node-fetch'
-import {
-  COUNTRY_CONFIG_HOST,
-  GATEWAY_HOST,
-  OPENCRVS_SPECIFICATION_URL
-} from './constants'
-import { TypeOf, z } from 'zod'
+import { COUNTRY_CONFIG_HOST, GATEWAY_HOST } from './constants'
+import { z } from 'zod'
 import { raise } from './utils'
 import { inspect } from 'util'
+import {
+  SavedBundle,
+  Location,
+  findStatisticalId,
+  SavedLocation
+} from '@opencrvs/commons/types'
+import chalk from 'chalk'
 
-const LocationSchema = z.array(
-  z.object({
-    id: z.string(),
-    name: z.string(),
-    alias: z.string().optional(),
-    partOf: z.string(),
-    locationType: z.enum(['ADMIN_STRUCTURE', 'HEALTH_FACILITY', 'CRVS_OFFICE']),
-    jurisdictionType: z
-      .enum([
-        'STATE',
-        'DISTRICT',
-        'LOCATION_LEVEL_3',
-        'LOCATION_LEVEL_4',
-        'LOCATION_LEVEL_5'
-      ])
-      .optional(),
-    statistics: z
-      .array(
-        z.object({
-          year: z.number(),
-          male_population: z.number(),
-          female_population: z.number(),
-          population: z.number(),
-          crude_birth_rate: z.number()
-        })
-      )
-      .optional()
-  })
-)
+const LocationSchema = z.object({
+  id: z.string(),
+  name: z.string(),
```

Comment:
updated this validation to be only the object instead of the array to infer the pure 1 item type simpler